manifests/kubeweekly-jobs.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kubeweekly-jobs
+spec:
+  schedule: "*/12 * * * 3"
```

Comment:
we will compile reading lists every weekend (bacaan akhir pekan)

Reply:
Probably add coments on what is this cron is doing

Reply:
addressed here https://github.com/cloudnative-id/kubeops/pull/3/commits/e8f6b264a502f199cea8f65e4b8c92bb2a70868f